src/Pim/Bundle/CatalogBundle/tests/integration/Elasticsearch/AbstractIndexConfigurationIntegration.php

```diff
@@ -424,9 +181,19 @@ private function getProductIndexConfiguration()
                 ],
                 'settings' => [
                     'analysis' => [
-                        'normalizer' => [
-                            'lowercase_normalizer' => [
-                                "filter" => ["lowercase"],
+                        'normalizer'  => [
+                            'text_normalizer'    => [
+                                'type'        => 'custom',
+                                'filter'      => ['lowercase'],
+                                // TODO: Should be ["html_strip", "newline_pattern"] ?
+                                // But is not supported for now in ES 5.2 even though it is documented.
+                                // Those string transformations can be made at PRE_INDEX time in PHP.
+                                // Normalizer is an experimental feature.
+                                // see https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-normalizers.html
+                                'char_filter' => [],
```

Comment:
Drawing your attention here

I searched for a way to duplicate the data in ES while transforming it at the same time with the operations: "html_strip" and "newline_pattern". Two things:

First, the new experimental normalizer feature should be able to do it (does not work for now, see documentation)
Second I found the ES Scripts (which use is discourage) but could do the job (I didn't find any html_strip function in the painless reference (maybe I missed something https://www.elastic.co/guide/en/elasticsearch/reference/master/painless-api-reference.html)
Maybe a simpler way would be to do it at PRE index time in the php side as already discussed with @jjanvier and @bitone (until the char_filter property is correctly implemented in ES ?)
I'm open to ideas.